.github/workflows/scripts/kubernetes/core/overridable-functions.sh

```diff
@@ -43,12 +46,53 @@ function installPrerequisites()
 #
 # Parameters
 # 1 - application name
+#
 function cleanPrerequisites()
 {
   application="${1}"
   echo "No prerequisites to clean for ${application}"
 }
 
+# Trigger the custom behaviour when it comes to
+# provision the server and push the imagestream.
+# Returns
+# 0 - false
+# 1 - true
+#
+function customProvisionServer() {
+  echo 0
+}
+
+# Provision server and push imagestream
+# The current directory is the quickstart directory
+#
+# Parameters
+# 1 - application name
+# 2 - quickstart dir
+#
+function provisionServer()
+{
+  echo "Nothing to do in provisionServer()..."
+}
+
+# Trigger a custom behaviour when it comes to
+# setting up the environment
+# Returns
+# 0 - false
+# 1 - true
+#
+function customProcessing() {
```

Comment:
'processing' feels a bit vague, and 'helmInstall' is already used. Maybe 'customHelmProcessing'?

Reply:
At least for `ejb-tx-remote-call`, this function is not about Helm. That's why I went for the generic `processing`. I'm very happy to go for something else (as I don't like `processing` myself) but I'd like something without helm in the function's name. WDYT?

Reply:
Hmm, maybe something like 'customDeploy'? I think this has the logic of how we deploy the app, and covers both the Helm and non-Helm cases?

Reply:
Cool. `deploy` and `customDeploy` will be then :-) Thanks Kabir